podcast/2.15.md

```diff
@@ -0,0 +1,1299 @@
+
+**Sugu Sougoumarane:**
+
+Our initial round was not by them, but they did plant the idea. And we did talk to them. We told them "You told us three years ago and here I am."
+
+**Beyang Liu:**
+
+Yeah. Okay. So that got you kind of thinking about the...
+
+**Sugu Sougoumarane:**
+
+Kind of. It was in the back of my mind.
+
+I was even willing to encourage other people to start a company and say, "If you start a company, I'll help you." From within YouTube. I don't care.
+
+**Beyang Liu:**
+
+Wow.
+
+**Sugu Sougoumarane:**
+
+But nobody did. We felt that for YouTube to continue to use Vitess, it had to become a mainstream project in its own right. So that was one motivation.
+
+**Beyang Liu:**
+
+Yeah. Keep it around.
+
+**Sugu Sougoumarane:**
+
+Yeah. But then there were other motivations. The other motivation was eventually what is happening is, YouTube was also changing as a company, where the newer and newer talent that was coming in, they were all product-oriented.
+
+Where does YouTube make money? YouTube does not make money by building Vitess.
+
+YouTube makes money by producing videos, publishing them, and stuff. That's where, naturally, all the effort was going. And at some point in time, the newer management in YouTube was starting to ask, why are we funding this project? But then this project is gaining adoption outside. Everybody is using it. We need this to be funded.
+
+And then there is Spanner, and Google internally was saying, "We don't know what this Vitess is."
+
+**Beyang Liu:**
+
+Just put it all in Spanner.
+
+**Sugu Sougoumarane:**
+
+Put it all in Spanner. That is what is blessed within Google. You should move to that. So I started talking to people. I talked to Joe Beda and Craig McLuckie, who were the founders of Heptio. So I started talking to him, because they kind of went through the same thing. They created Kubernetes, and ended up leaving YouTube to start Heptio as a company. And they also founded the CNCF.
+
+**Beyang Liu:**
+
+Oh, I did not know that. Interesting.
+
+**Sugu Sougoumarane:**
+
+They had done all that, so we kind of chatted with them. "So Vitess is in this stage where YouTube doesn't want to spend more money on it. How did you manage Kubernetes?" That's when they said, "Vitess should join CNCF. It's the perfect fit." They kind of encouraged us to do it. So I went back and proposed to YouTube that we should donate Vitess to CNCF. YouTube liked the idea. Said it's perfect. And the kind of understanding was that after that I will go and start a company on Vitess. That's kind of how this all happened. So this happened during 2017. And in 2018, we started the company.
+
+**Beyang Liu:**
+
+So, the company is quite young, even though Vitess has been around.
+
+**Sugu Sougoumarane:**
+
+Yes. Yes. That was actually what the community was missing. Because Vitess is not not a tooling that you try today, you don't like it, you throw it out tomorrow. It stores your core data, your core infrastructure.
```

Comment:
```suggestion
Yes. Yes. That was actually what the community was missing. Because Vitess is not a tooling that you try today, you don't like it, you throw it out tomorrow. It stores your core data, your core infrastructure.
```